### PR TITLE
Issue 5 - RU translation

### PR DESCRIPTION
**Pull request by ftvkyo**

### CODE REVIEW

**Review by Firerazer — CHANGES_REQUESTED** [review]
File structure is lost. Please adjust accordingly to source files.
No other checks have been done, waiting for file structure to be fixed.

**Comment by ftvkyo** [comment]
Corrected the file structure for the paragraphs to follow the original style of one line per paragraph, links defined in-text.
Lines are offset by one because of `translator:ftvkyo` added to the metadata yaml.

File: _posts/ru/2020-09-18-ALTF4-5.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+author: pocarski, Romner, Dr.MagPie
+editor: Therenas, nicgarner
+translator: ftvkyo
+image:
+  url: /assets/ALTF4/5/thumbnail.jpg
+  transcription: Fusion rocket exploding
+title: "Alt-F4 #5 - Improving on Perfection"
```
Comment (Firerazer) [inline _posts/ru/2020-09-18-ALTF4-5.md]: Line 7-8 need translation

File: _posts/ru/2020-09-18-ALTF4-5.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+lang: en
```
Comment (Firerazer) [inline _posts/ru/2020-09-18-ALTF4-5.md]: this require to be changed to "ru" to be referred to the correct language

**Review by Firerazer — CHANGES_REQUESTED** [review]

File: _posts/ru/2020-09-18-ALTF4-5.md (hunk 1 above)
Comment (ftvkyo) [reply _posts/ru/2020-09-18-ALTF4-5.md]: I see, thanks. When I looked for a reference I happened to look at posts that didn't translate these lines. I'll translate these.

Comment (Firerazer) [reply _posts/ru/2020-09-18-ALTF4-5.md]: correct, some posts in russian seems to lack these being translated. you can work on it if you want :)

**Pull request merged**